server/opts.go

```diff
@@ -489,12 +492,67 @@ func (o *Options) ProcessConfigFile(configFile string) error {
 				continue
 			}
 			o.LameDuckDuration = dur
-		case "trusted":
+		case "operator", "operators", "roots", "root", "root_operators", "root_operator":
+			opFiles := []string{}
+			switch v.(type) {
+			case string:
+				opFiles = append(opFiles, v.(string))
+			case []string:
+				opFiles = append(opFiles, v.([]string)...)
```

Comment:
Should you have a `default:` here and report error (like you do for resolver)?

Reply:
yes will fix.